sources/packages/forms/src/form-definitions/sfaa2025-26-ft.json

```diff
@@ -4100,159 +4171,166 @@
-                            {
-                              "attr": "",
-                              "value": ""
-                            }
-                          ],
-                          "content": "<h4 class=\"category-header-medium-small\">Instructions:</h4>\n<ul>\n  <li>Upload a photo or scanned copy of your documentation</li>\n  <li>Rename your document to <br/>\"<strong>FirstnameLastname_DependantsFirstNameLastName_TaxYear</strong>\"</li>\n  (e.g. JessLee_AlexLeeTax_2025)\n</ul>\n",
-                          "refreshOnChange": false,
-                          "customClass": "align-bullets",
-                          "key": "html38",
-                          "type": "htmlelement",
-                          "input": false,
-                          "tableView": false
-                        },
-                        {
-                          "label": "PD Dependent",
-                          "tableView": false,
-                          "storage": "url",
-                          "dir": "PD Dependent",
-                          "webcam": false,
-                          "fileTypes": [
+                          "components": [
                             {
-                              "label": "",
-                              "value": ""
-                            }
-                          ],
-                          "filePattern": ".pdf,.doc,.docx,.jpg,.png,.txt",
-                          "fileMaxSize": "15MB",
-                          "multiple": true,
-                          "redrawOn": "dependants.declaredOnTaxes",
-                          "validate": {
-                            "required": true,
-                            "customMessage": "You must upload at least one file, and the total number of uploaded files cannot exceed {{data.maxUploadedFiles}}.",
-                            "custom": "const files = data[instance.key] || []; valid = files.length > data.maxUploadedFiles ? false : true;"
-                          },
-                          "key": "pdDependentUpload",
-                          "type": "file",
-                          "url": "student/files",
-                          "input": true,
-                          "conditional": {
-                            "show": "true",
-                            "when": "declaredOnTaxes",
-                            "eq": "yes"
-                          }
-                        },
-                        {
-                          "input": false,
-                          "key": "columns8",
-                          "tableView": false,
-                          "label": "Columns",
-                          "type": "columns",
-                          "columns": [
+                              "label": "Exception Description - Dependant's income tax",
+                              "calculateValue": "value = `Dependant's income tax - ${data.dependants[rowIndex].fullName}`;",
```

Comment:
Will `row.fullName` name work instead?

Reply:
Apparently, the container adds some isolation that makes this not work.